_data/rules.yml

```diff
@@ -1105,7 +1105,21 @@ groups:
                 query: 'sum(kafka_consumergroup_lag) by (consumergroup) > 50'
                 severity: critical
                 for: 1m
-
+          - name:  linkedin/Burrow 
+            doc_url: https://github.com/linkedin/Burrow
+            rules:
+              - name: Kafka Offset Decreased
+                description: Kafka topic offset was decreased
+                query: 'delta(kafka_burrow_partition_current_offset[1m])<0'
+                severity: high
+                for: 3m
```

Comment:
Does it make sense to wait 3min for this alert to trigger? 🤔

Reply:
Sounds reasonable